examples/computer_vision/efficientdet_pytorch/adaptive.yaml

```diff
@@ -0,0 +1,104 @@
+description: efficientdet_adaptive_loss
+hyperparameters:
+  global_batch_size: 32
+  min_loss_scale:
+    type: double
+    minval: 8
+    maxval: 64
+  sync_bn: 
+    type: categorical
+    vals: [True, False]
+  model_ema: True
+  amp: True
+  root: '/mnt/data'
+  dataset: coco
+  model: efficientdet_d0
+  redundant_bias: None
+  val_skip: 0
+  num_classes: None
+  pretrained: False
+  no_pretrained_backbone: False
+  model_ema_decay: 0.9999
+  initial_checkpoint: None
+  dist_bn: None
+  output: None
+  decay_rate: 0.1
+  resume: ''
+  no_resume_opt: False
+  mean: None
+  std: None
+  interpolation: ''
+  fill_color: None
+  clip_grad:
+    type: double
+    minval: 5.0
+    maxval: 10.0
+  opt:
+    type: categorical
+    vals: ['fusedmomentum','adam', 'momentum', 'adamw']
+  opt_eps: 0.001
+  momentum: 0.9
+  weight_decay: 
+    type: double
+    minval: .00004
+    maxval: .002
+  sched: 
+    type: categorical
+    vals: ['cosine', 'tanh', 'step']
+  lr:
+    type: double
+    maxval: 0.3
+    minval: 0.001
+  lr_noise: 0.4 0.9
+  lr_noise_pct: 0.67
+  lr_noise_std: 1.0
+  lr_cycle_mul: 1.0
+  lr_cycle_limit: 1
+  warmup_lr: 0.0001
+  min_lr: 1e-5
+  start_epoch: None
+  decay_epochs: 30
+  warmup_epochs: 5
+  cooldown_epochs: 10
+  patience_epochs: 10
+  color_jitter: 0.4
+  aa: None
+  reprob: 0.
+  remode: pixel
+  recount: 1
+  train_interpolation: random
+  smoothing: None
+  jit_loss: None
+  new_focal: None
+  seed: 42
+  recovery_interval: 0
+  workers: 4
+  save_images: False
+  channels_last: False
+  pin_mem: False
+  no_prefetcher: False
+  bench_labeler: False
+  eval_metric: map
+  tta: 0
+  epochs: 300 # used for LR calculations:update according to max_length
+records_per_epoch: 118287
+searcher:
+  name: adaptive_asha
+  metric: val_loss
+  max_length:
+      epochs: 300
+  smaller_is_better: False
+  max_trials: 1000
+  max_concurrent_trials: 4
+resources:
+  slots_per_trial: 2
+reproducibility:
+  experiment_seed: 42
+optimizations:
+   tensor_fusion_threshold: 1
+   tensor_fusion_cycle_time: 2
+entrypoint: model_def:EffDetTrial
+max_restarts: 0
+bind_mounts:
+  - host_path: /home/katie/projects/data/coco
```

Comment:
Can you put a comment here basically asking the user to replace this path with their path to COCO and perhaps mention that in the README somehow or something?